### PR TITLE
Enable platform-specific CAs, including ARM64

### PR DESCRIPTION
**Pull request by barnson**
- Update to latest dependencies.

### CODE REVIEW

File: src/ca/netfxca.vcxproj [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
 <Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\..\packages\WixToolset.WcaUtil.4.0.11\build\WixToolset.WcaUtil.props" Condition="Exists('..\..\packages\WixToolset.WcaUtil.4.0.11\build\WixToolset.WcaUtil.props')" />
-  <Import Project="..\..\packages\WixToolset.DUtil.4.0.24\build\WixToolset.DUtil.props" Condition="Exists('..\..\packages\WixToolset.DUtil.4.0.24\build\WixToolset.DUtil.props')" />
+  <Import Project="..\..\packages\WixToolset.DUtil.4.0.26\build\WixToolset.DUtil.props" Condition="Exists('..\..\packages\WixToolset.DUtil.4.0.26\build\WixToolset.DUtil.props')" />
+  <Import Project="..\..\packages\WixToolset.WcaUtil.4.0.13\build\WixToolset.WcaUtil.props" Condition="Exists('..\..\packages\WixToolset.WcaUtil.4.0.13\build\WixToolset.WcaUtil.props')" />
```
Comment (rseanhall) [inline src/ca/netfxca.vcxproj:5]: This makes me wonder if you are aware of the script I wrote to update package versions - https://github.com/wixtoolset/Home/blob/c9ea8634b3ea512aa244efe6024f6160b27ebcab/updatepackage.ps1

Comment (barnson) [reply src/ca/netfxca.vcxproj:5]: PowerShell. Yuck.

Comment (rseanhall) [reply src/ca/netfxca.vcxproj:5]: Well how are you doing it then? Pushing buttons in VS? So pedestrian.

Comment (barnson) [reply src/ca/netfxca.vcxproj:5]: Notepad. Well, technically Notepad++. That lets me rearrange for the proper order of DUtil, thjen WcaUtil.

**Pull request merged**